static/js/containers/ProfileFormContainer.js

```diff
@@ -112,15 +113,19 @@ class ProfileFormContainer extends React.Component {
     dispatch(startProfileEdit(username));
   };
 
-  saveProfile(isEdit: boolean, validator: Validator|UIValidator, profile: Profile, ui: UIState) {
+  saveProfile(isEdit: boolean, validator: Validator|UIValidator|null, profile: Profile, ui: UIState) {
```

Comment:
I don't understand: the function is called `saveProfile`, implying that the validator should be for the profile as a whole, not for an individual field. Surely there would be a validator for the profile as a whole, even if that validator skips over an individual field in the profile. Is this function poorly named? Or am I misunderstanding how this works?